pkg/clusteragent/autoscaling/externalmetrics/datadogmetric_controller.go

```diff
@@ -135,16 +151,25 @@ func (c *DatadogMetricController) enqueueID(id, sender string) {
 	}
 }
 
-func (c *DatadogMetricController) process() bool {
+func (c *DatadogMetricController) process(workerID int) bool {
```

Comment:
can we attach to the logger a workerID to easily differentiate the different process() execution path?

Reply:
I think I will bring this limitation to the team owning the logger.
Indeed I was thinking of the logger we have in the datadog-operator.
But I was thinking that now structured logging was available also in the agent.

Reply:
It is possible but you need to propagate the info to put in the structured logger down (as no sublogger)